src/components/Session/TrustButton.jsx

```diff
@@ -0,0 +1,61 @@
+const React = window.React = require('react');
+import _ from 'lodash';
+
+export default class TrustButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      status: 'ready', // ready, error, or pending
+      errorType: '', // 'unknown' | 'lowReserve'
+    }
+  }
+
+  _handleSubmitTrust(event) {
```

Comment:
No need for underscores because we never call anything from a react component.

Also, can you put this back into the constructor with the fat arrow `()=>{}` function so that theres less binding happening?

Thanks

Reply:
ok, I'll move this (idiomatic React code discourages having any data on the `this` object).

Reply:
also, the fat arrow is implicitly binding under the covers anyways :)

Reply:
The implicit binding is nice because it reduces overhead. By just using fat arrows, we almost never have to worry about binding. With having a class function, if someone is tired, they might forget to use `bind` and then have trouble figuring out a bug.

I prefer to have simplicity and ease of coding over following dogma.